src/main/java/plugin/sample/Main.java

```diff
@@ -86,7 +86,7 @@ public void onPlayerBedEnter(PlayerBedEnterEvent e) {
     ItemStack[] itemStacks = player.getInventory().getContents();
     Arrays.stream(itemStacks).filter(
             item -> !Objects.isNull(item) && item.getMaxStackSize() == 64 && item.getAmount() < 64)
-        .forEach(item -> item.setAmount(64));
+        .forEach(item -> item.setAmount(32));
```

Comment:
寝たときのアイテム数を変更

Reply:
```suggestion
        .forEach(item -> item.setAmount(32));
```
ここで取得したアイテムをfilterの条件に該当した場合アイテム数をセットする